engines/config-query-sparql/config/rdf-join/actors.json

```diff
@@ -3,8 +3,7 @@
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/config-query-sparql/^3.0.0/components/context.jsonld"
   ],
   "import": [
-    "ccqs:config/rdf-join/actors-inner.json",
-    "ccqs:config/rdf-join/actors-minus.json",
+    "ccqs:config/rdf-join/actors-base.json",
```

Comment:
I would not introduce an actors-base file, it's clearer IMO to just have the imports directly.